kubernetes/samples/scripts/create-soa-domain/domain-home-on-pv/common/create-domain-job.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
-# Copyright (c) 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
-# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+# Copyright 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
```

Comment:
Everything else looks good, but the copyrights (as Mark noted) are incorrect in this change.  We got pushback from CorpArch clarifying the expectations and you can review them yourself at the link.

Reply:
Fixed Copyright statements in all the scripts.
Do we need to add the same to below yaml configuration files/templates ? Please suggest.

create-soa-domain/domain-home-on-pv/**wdt**/wdt_model_configured.yaml
create-soa-domain/domain-home-on-pv/**wdt**/wdt_model_dynamic.yaml         
create-soa-domain/domain-home-on-pv/**wdt**/wdt_model_restricted_jrf_configured.yaml
                            
create-soa-domain/domain-home-on-pv/**create-database**/db-with-pv.yaml
create-soa-domain/domain-home-on-pv/**create-database**/db-without-pv.yaml

Reply:
Hi Mark,
Could you please suggest on this ? so that I can update the files if any changes required.

Thanks
Samba

Reply:
We have another set of scripts to start ans stop oracle db service in the sample directory kubernetes/samples/scripts/create-rcu-schema. The scripts are (I) start-db-service.sh and (II) stop-db-service.sh. We are using the scripts in integration tests to run FMW Domain test. 
 
I  can move this scripts to one level up to kubernetes/samples/scripts/create-oracle-db so that it can be used by both SOA and FMW domain and avoid duplicate scripts (create-soa-domain/create-database) doing the same.

Reply:
yes you should add copyright statements to all files, unless there is some specific reason why it is not possible, e.g. json does not support comments